library/jruby

```diff
@@ -1,22 +1,22 @@
 Maintainers: Brian Goff <cpuguy83@gmail.com> (@cpuguy83)
 GitRepo: https://github.com/cpuguy83/docker-jruby.git
 GitFetch: refs/heads/master
-GitCommit: 4a733532d5cdffdbafd96be19e2e9862bfe61056
+GitCommit: a9179975e60a96a6e9d880bcf2dd566ab290b539
 
-Tags: latest, 9, 9.1, 9.1-jre, 9.1.13, 9.1.13-jre, 9.1.13.0, 9.1.13.0-jre
+Tags: latest, 9, 9.1, 9.1.13, 9.1-jre, 9.1.13-jre, 9.1.13.0, 9.1.13.0-jre
 Architectures: amd64, i386, arm32v5, arm64v8, ppc64le, s390x
 Directory: 9000/jre
 
-Tags: 9-alpine, 9.1-alpine, 9.1-jre-alpine, 9.1.13-alpine, 9.1.13-jre-alpine, 9.1.13.0-alpine, 9.1.13.0-jre-alpine
-Architectures: amd64, arm32v6, arm64v8, ppc64le, s390x
+Tags: 9-alpine, 9.1-alpine, 9.1.13-alpine, 9-alpine, 9.1-jre-alpine, 9.1.13-jre-alpine, 9.1.13.0-alpine, 9.1.13.0-jre-alpine
```

Comment:
`9-alpine` is in here twice :smile:

Reply:
Also another issue, but you'll never find it now :)